### ISSUE TITLE
provide state: latest in manala_apt_packages

### ISSUE THREAD

**Issue by raleonardo** [manala/ansible-roles#209]
provide state: latest in manala_apt_packages

**Comment by nervo** [comment]
Hm, i'm afraid this is not possible with our current implementation, as packages are bunk installed, so that individual states could not be handled (see: https://github.com/manala/ansible-roles/blob/master/manala.apt/tasks/packages.yml#L16)

Maybe if you have some idea ?..

**Comment by raleonardo** [comment]
Thanks for the comment! I understand that packages listed for present are installed in bulk.
I added another task there matching wantstate=latest and calling apt with state=latest.

**Comment by nervo** [comment]
@raleonardo what about a global flag ? something like `manala_apt_packages_latest: true|false`
Do you want only some packages in latest state, or all of them ?

**Comment by raleonardo** [comment]
Yep. Although per package would be ideal, global could be also useful! :)

--
Ricardo Leonardo
